docs/examples.md

```diff
@@ -15,3 +15,22 @@ python examples/benchmark_evaluator.py --push-to-db --model=<the model used to b
 
 ## Example 2: Generate script-like episodes
 See `docs/simulation_modes.md` for more information.
+
+## Example 3: Benchmarking the models as social agents
+
+```Bash
+EVAL_MODEL="gpt-4o-2024-05-13"
```

Comment:
This doc seems to be inconsistent with the API. Would you mind git merge main and then add a new page on benchmarking?